src/main/java/tech/jhipster/controlcenter/config/SecurityConfiguration.java

```diff
@@ -85,21 +86,16 @@ public SecurityWebFilterChain springSecurityFilterChain(ServerHttpSecurity http)
             .pathMatchers("/").permitAll()
             .pathMatchers("/*.*").permitAll()
             .pathMatchers("/api/auth-info").permitAll()
-            // jhcc-custom : need to be deleted -- begin
             .pathMatchers("/api/authenticate").permitAll()
-            .pathMatchers("/api/**").permitAll()
+            .pathMatchers("/api/**").authenticated()
+            // jhcc-custom : begin
+            .pathMatchers("/services/**", "/gateway/**", "/v2/api-docs", "/swagger-ui/index.html").authenticated()
```

Comment:
I have doubt about  `"/v2/api-docs", "/swagger-ui/index.html"` paths. Do we need to put them under an authenticated matcher?
Do we have any reasons to force people to be connected to reach the api doc? 

For me we don't need to force to be authenticated to do that but I could be wrong. WDYT @pascalgrimaud @PierreBesson

Reply:
this is what we have when we generate an application with JHCC .yo-rc 
`/services/**", "/swagger-resources/**", "/v2/api-docs"`

Reply:
We put "/v2/api-docs" and "/swagger-ui/index.html" under permitAll by default in the next commit.